psychopy/app/localization/__init__.py

```diff
@@ -127,7 +128,12 @@ def getID(lang=None):
 except IOError:
     logging.debug("Locale for '%s' not found. Using default." % lang)
     trans = gettext.NullTranslations()
-trans.install()
+
+# gettext.install() needs unicode=True to get unicode output in Python2.
+if sys.version_info[0] >= 3:
```

Comment:
Can you please use [`psychopy.constants.PY3`](https://github.com/psychopy/psychopy/blob/9fb5dd91767b3ad112e47a529ac8cc9e1f0254cd/psychopy/constants.py#L11) instead? Thanks!